ui/lib/sync/addon/components/secrets/page/overview.ts

```diff
@@ -74,10 +74,11 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
   @task
   @waitFor
   *onFeatureConfirm() {
+    const namespace = this.featureFlag.managedNamespaceRoot;
```

Comment:
To clarify this will either return null for root or admin for HCP?

Reply:
That's correct